language-service/tests/service/definition/wiktionary/test_definition.py

```diff
@@ -0,0 +1,54 @@
+import pytest
+from testfixtures import compare
+from unittest.mock import Mock
+from language_service.dto import Definition
+from language_service.service.definition.wiktionary import Wiktionary
+
+
+def test_can_fetch_definitions(mocker):
+    wiktionary = Wiktionary()
+    mocker.patch.object(wiktionary, "fetch")
+
+    definition = wiktionary.get_definitions("CHINESE", "定义")
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [local variable 'definition' is assigned to but never used](https://app.codacy.com/manual/lucaskjaero/foreign-language-reader/pullRequest?prid=5124786)